.github/workflows/Publish.yml

```diff
@@ -1,49 +1,21 @@
-name: BuildTestPublish
+name: Publish
 
 on:
-  workflow_dispatch:
   workflow_call:
+    secrets:
+      GCR_PASSWORD:
+        description: "Password for Github Container Registry"
```

Comment:
Hva gjør denne/trengs det for? 👀 
Docker-layer-caching?

Reply:
Nei det er bare jeg som renama GITHUB_TOKEN til GCR_PASSWORD. Mulig det gjør det mer confusing altså, men tenkte at GCR_PASSWORD som i Github Container Registry password egnet seg bedre. Vi bruker GITHUB_TOKEN som verdi der, men etterhvert så kanskje vi endrer det til noe annet. (mest sannsynlig ikke tho..)

Reply:
Hva tenker du? Kan også rename til GITHUB_TOKEN begge steder altså

Reply:
Aah, sånn ja.
Nå skjønner jeg, har du da glemt å bruke denne secreten i denne fila? 
Er vel fortsatt `password: ${{ secrets.GITHUB_TOKEN }}` nedi matrisa her

Reply:
oh haha, yeah glemte å endre der ja. Nice spot 😄